src/frontend/store/OnboardingStore.tsx

```diff
@@ -13,6 +13,9 @@ class OnboardingStore {
 
   constructor() {
     makeAutoObservable(this)
+    window.api.handleOpenOnboarding(() => {
+      this.openOnboarding()
+    })
```

Comment:
we need this to be able to trigger opening the modal in the main window from another.